Source/Core/Core/PowerPC/JitArm32/JitArm_Paired.cpp

```diff
@@ -21,7 +21,6 @@ void JitArm::ps_rsqrte(UGeckoInstruction inst)
 {
 	INSTRUCTION_START
 	JITDISABLE(bJITPairedOff);
-	FALLBACK_IF(true);
 
```

Comment:
Using bad approximations of rsqrte is known to cause issues.  See https://dolphin-emu.org/blog/2014/05/30/dolphin-progress-report-may-2014/ .